docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -67,6 +67,16 @@ Throttling can also be disabled in the configuration file using
 <<synthetics-configuration-monitor,`monitor.throttling`>>.
 The value defined via the CLI will take precedence.
 
+`--no-headless`::
+Runs with the browser in headful mode.
++
+This is the same as running `--playwright-options '{"headless": false}'`.
+
+[NOTE]
+=====
+Headful mode is only to be used locally when testing, to help see the browser being shown and to interact with DOM elements. Do not attempt to run in headful mode when running through Elastic's global managed testing infrastructure, or {private-location}s.
```

Comment:
```suggestion
Headful mode should only be used locally to see the browser and interact with DOM elements directly for testing purposes. Do not attempt to run in headful mode when running through Elastic's global managed testing infrastructure or {private-location}s.
```

Reply:
Can we also mention that we dont support on our infrastructure instead of saying not to use it.